lua/entities/gmod_tardis/modules/cl_effects.lua

```diff
@@ -0,0 +1,46 @@
+local function rand_offset()
+	return math.random(-35, 35)
+end
+
+local function get_effect_pos(self)
+	local console = self.interior:GetPart("console")
+	if self.metadata.Interior.BreakdownEffectPos then
+		self.effect_pos = self.interior:GetPos() + self.metadata.Interior.BreakdownEffectPos
+	elseif console and IsValid(console) then
+		self.effect_pos = console:GetPos() + Vector(0, 0, 50)
+	else
+		self.effect_pos = self.interior:GetPos() + Vector(0, 0, 50)
+	end
+end
+
+function ENT:InteriorExplosion()
+	if self.effect_pos == nil then
+		get_effect_pos(self)
+	end
+
+	local function rand_offset() return math.random(-40, 40) end
```

Comment:
`rand_offset` function appears twice with slightly different values, this should be consolidated into the one function. If you need to pass different parameters e.g. 35 vs 40 make it a parameter to the function